packages/eslint-config-loris/README.md

```diff
@@ -21,7 +21,7 @@ module.exports = {
 };
 ```
 
-* For [ECMAScript 6](https://www.ecma-international.org/ecma-262/6.0/index.html)
+* For [ECMAScript 2015](https://www.ecma-international.org/ecma-262/6.0/index.html)
```

Comment:
Чтобы путаницы не было, может стоит `loris/es6` переименовать в `loris/es2015` и добавить симлинк для обратной совместимости, пометив его deprecated?

Reply:
Since this is new major release, we can just drop `es6` preset. Actually, I don't think that is matter, because `es6` and `es2015` are interchangeable from my point of view. Anyway, this is separated issue/pull request.

Reply:
I decided to not introduce breaking changes in new version. So, new version is `9.1.0`.